README.md

```diff
@@ -1,18 +1,27 @@
 # Tapas client Javascript
 
+## TL;DR
+
+TODO
+
 ## Init
 
-```js
-let tapas = new Tapas()
-```
+The Tapas client communicates via the Tapas server via a unique endpoint.
+All the queries are sent to this unique endpoint. But don't worry, the Tapas
+Client is will manage all the communications for you. You'll just have to
+concentrate on the data you need.
+
+To start working with Tapas, create a new instance of `Tapas`, providing the
+name of the transport protocol and the address of the endpoint. The supported
+transport protocols are `http` and `websocket`. 
 
 ```js
-let tapas = new Tapas('websocket', {port: 8080})
+let tapas = new Tapas('websocket', {endpoint: 'http://localhost:7513'})
```

Comment:
`websocket` looks like an option. We probably have to put this at the end of the function? Something like this:
`let tapas = new Tapas('localhost:7513', 'websocket')` or directly `let tapas = new Tapas('ws://localhost:7513')`. I don't know